src/server_manager/web_app/testing/models.ts

```diff
@@ -127,6 +127,9 @@ export class FakeServer implements server.Server {
   getVersion() {
     return '1.2.3';
   }
+  getAccessKey(_accessKeyId: server.AccessKeyId) {
```

Comment:
This is not being used in the Manager. Please revert the server manager changes.

Reply:
Done.

Reply:
I still see the Manager changes there: 
![image](https://user-images.githubusercontent.com/113565/186745643-716fa27b-e777-441f-ad51-5fe72fa46c7c.png)

Please revert those.